rasa/nlu/classifiers/sklearn_intent_classifier.py

```diff
@@ -165,49 +182,51 @@ def _create_classifier(
             verbose=1,
         )
 
-    def process(self, message: Message, **kwargs: Any) -> None:
+    def process(self, messages: List[Message]) -> List[Message]:
         """Return the most likely intent and its probability for a message."""
+        for message in messages:
+            if not self.clf:
+                # component is either not trained or didn't
+                # receive enough training data
+                intent = None
+                intent_ranking = []
```

Comment:
should we pull this out of the loop?

Reply:
And then return the list with unmodified messages?

Reply:
mhm, actually that makes sense 👍🏻 Let's keep it in then